src/ReflectionFile.php

```diff
@@ -50,6 +50,9 @@ class ReflectionFile
      */
     public function __construct($fileName, $topLevelNodes = null)
     {
+        if (!is_string($fileName)) {
```

Comment:
Then better to:

1. revert this change
2. in another PR address cases, when method can be called with non-string argument